docs/cloud-native-security/security-posture-management.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[security-posture-management]]
+= Security posture management
+
+[discrete]
+== Overview
+The <<cspm,Cloud Security Posture Management>> (CSPM) and <<kspm,Kubernetes Security Posture Management>> (KSPM) integrations discover and evaluate the services and resources in your cloud environment — like storage, compute, IAM, and more — against security guidelines defined by the Center for Internet Security (CIS). They help you identify and remediate configuration risks that could undermine the confidentiality, integrity, and availability of your cloud assets, such as publicly exposed storage buckets or overly permissive networking objects.
+
+The KSPM integration can assess the security of your Kubernetes assets, while the CSPM integration assesses the security of your AWS resources such as storage, compute, IAM, and more.
+
+[discrete]
+== Getting started
+For setup instructions, refer to:
+
+* <<get-started-with-kspm, Get started with KSPM>>
+* <<cspm-get-started, Get started with CSPM>>
+
+[discrete]
+[[security-posture-use-cases]]
+== Use cases
+
+Using the data generated by these integrations, you can:
+
+*Identify and secure misconfigured infrastructure:*
+
+. Go to the Cloud Posture dashboard (*Dashboards > Cloud Posture*).
+. Click *View all failed findings*, either for an individual resource or a group of resources.
+. Click a failed finding to open the Findings flyout.
+. Follow the steps under Remediation to fix the misconfiguration.
+
+*Identify the CIS Sections (security best practice categories) with which your resources are least compliant:*
+
+. Go to the Cloud Posture dashboard (*Dashboards > Cloud Posture*).
+. Under Failed findings by CIS section, click the name of a CIS section to view all failed findings from that section.
+
+Alternatively:
```

Comment:
This sticks out a bit. What do you think about combining this with step 2? Maybe something like:

2. Choose one of the following:
 - Under Failed findings by CIS section, click the name of a CIS section to view all failed findings from that section.
 - Go to the Findings page and filter by the rule.section field. For example, search for rule.section : API Server to view findings from the API Server category.

Reply:
💯 💯